code/mathematical-algorithms/Check_is_square/check_is_square.swift

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env swift
+// Part of Cosmos by OpenGenus Foundation
+import Darwin
+
+func findingSquare(num : Int) -> Int {
+    var res = 0
+    let x = sqrt(Double(num))
+    for i in 0...num {
```

Comment:
You could simplify it by casting `x` to `Int` and doing `for i in 0...x`, removing the conditional at line 13